release/pipline/dags/istio_common_dag.py

```diff
@@ -174,7 +177,15 @@ def GenerateTestArgs(**kwargs):
     git config --global user.email "testrunner@istio.io"
     git clone {{ settings.MFEST_URL }} green-builds || exit 2
     cd green-builds
-    git checkout {{ settings.MFEST_COMMIT }} || exit 3
+    SHA=`grep {{ settings.MFEST_FILE }} | cut -f 6 -d \\"` || exit 3
+    cd ..
+    git clone {{ settings.ISTIO_REPO }} istio-code
+    cd istio-code/release
+    git checkout ${SHA} || exit 4
+    gsutil cp *.sh gs://{{ settings.GCS_RELEASE_TOOLS_PATH }}/data/release/
+    gsutil cp *.json gs://{{ settings.GCS_RELEASE_TOOLS_PATH }}/data/release/
+    cd ../../green-builds
```

Comment:
I tend to use pushd/popd more than backing out using "cd ..".  I tend to think of it as insulating me a bit more from unknown/unexpected changes made by others to the pwd, though the pitfall tradeoff is if someone accidently has a pushd without a matching popd.  Either way, I'm wondering whether this script block should be less surprising by returning to the original directory at the end, and I'm not clear on whether something should delete these repos so storage doesn't eventually get exhausted.